### PR TITLE
Merge 6.5-rc-2 into develop

### PR DESCRIPTION
**Pull request by jkmassel**
Merges changes from the `6.5-rc-2` release into `develop`.

### CODE REVIEW

File: WooCommerce/build.gradle [hunk 1]
```diff
@@ -44,9 +44,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "6.5-rc-1"
+            versionName "6.5-rc-2"
         }
-        versionCode 210
+        versionCode 212
```
Comment (loremattei) [inline WooCommerce/build.gradle:49]: Why did this jump?

**Review by loremattei — APPROVED** [review]
Left a question, but LGTM!

File: WooCommerce/build.gradle (hunk 1 above)
Comment (oguzkocer) [reply WooCommerce/build.gradle:49]: @loremattei It looks like there was a `6.4.1` hotfix https://github.com/woocommerce/woocommerce-android/pull/3876, so @jkmassel had to manually bump the version in https://github.com/woocommerce/woocommerce-android/pull/3878/commits/bd90f7845668f044d12e74be6f1ead54c77bb99e.

It all looks good to me. 🤷‍♂️

**Review by oguzkocer — APPROVED** [review]

File: WooCommerce/build.gradle (hunk 1 above)
Comment (jkmassel) [reply WooCommerce/build.gradle:49]: Yep that's right!

**Pull request merged**